2018-02-21 11:36:28,660 ERROR: Exception on /unvote/3 [GET] [in /usr/local/lib/python2.7/dist-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/broccoli/161220049_web_programming/blog/app/routes.py", line 180, in unvote
    if not post.vote_num>0:
AttributeError: 'Post' object has no attribute 'vote_num'
2018-02-21 11:36:30,043 ERROR: Exception on /unvote/3 [GET] [in /usr/local/lib/python2.7/dist-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/broccoli/161220049_web_programming/blog/app/routes.py", line 180, in unvote
    if not post.vote_num>0:
AttributeError: 'Post' object has no attribute 'vote_num'
2018-02-21 11:36:38,161 ERROR: Exception on /unvote/3 [GET] [in /usr/local/lib/python2.7/dist-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/broccoli/161220049_web_programming/blog/app/routes.py", line 180, in unvote
    if not post.vote_num>0:
AttributeError: 'Post' object has no attribute 'vote_num'
2018-02-21 11:36:45,430 ERROR: Exception on /unvote/2 [GET] [in /usr/local/lib/python2.7/dist-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/broccoli/161220049_web_programming/blog/app/routes.py", line 180, in unvote
    if not post.vote_num>0:
AttributeError: 'Post' object has no attribute 'vote_num'
2018-02-21 11:38:45,762 INFO: Blog startup [in /home/broccoli/161220049_web_programming/blog/app/__init__.py:44]
2018-02-21 11:45:04,437 INFO: Blog startup [in /home/broccoli/161220049_web_programming/blog/app/__init__.py:44]
2018-02-21 11:49:19,015 INFO: Blog startup [in /home/broccoli/161220049_web_programming/blog/app/__init__.py:44]
2018-02-21 11:57:43,457 INFO: Blog startup [in /home/broccoli/161220049_web_programming/blog/app/__init__.py:44]
